src/sipmfit.jl

```diff
@@ -34,7 +34,7 @@ function fit_sipm_spectrum(pe_cal::Vector{<:Real}, min_pe::Real=0.5, max_pe::Rea
     dmat = reshape(amps_fit, length(amps_fit), 1)
 
     # set up mixture model with given number of mixtures
-    gmm = GMM(n_mixtures, dmat; method=method, nInit=nInit, nIter=nIter, kind=kind)
+    gmm = GMM(n_mixtures, dmat; method=method, nInit=nInit, nIter=nIter, kind=kind, parallel=false)
```

Comment:
This requires https://github.com/davidavdav/GaussianMixtures.jl/pull/114 to be merged and released first.